### DISCUSSION TITLE
Two videos simultaneously

### DISCUSSION THREAD

**Discussion by SanjitRaman** [lagmoellertim/unsilence#154] [General]
I may typically have a camera of the lecturer and a screenshare of the slides, both as separate mp4 files. The screenshare has no audio.

Is there a way to apply the same unsilence to both files? 

Thanks

**Comment by ranagee007** [comment]
Yes, you can synchronize the audio from the lecturer's camera with the slides from the screenshare by merging the two separate MP4 files. You'll need video editing software to accomplish this task. Here's a general guide on how you might approach this using a common video editing tool like Adobe Premiere Pro or similar software:

Import Files:
Open your video editing software and import both the MP4 files into the project.

Arrange Tracks:
Drag the video track from the lecturer's camera and the screenshare slides onto the timeline. Ensure that they are aligned at the starting point.

Align the Tracks:
Find a distinct point in both videos where you can easily sync them. This might be a clap, a verbal cue, or any noticeable action. Use this point to align the two tracks.

Adjust Audio:
If the screenshare has no audio, you'll want to use the audio from the lecturer's camera. Mute or remove the original audio track from the screenshare video, and then align the audio from the camera with the corresponding visuals on the screenshare.

Cut and Adjust:
Trim or cut the tracks as necessary to remove any unnecessary parts, ensuring that the synchronized audio and video remain intact.

Export:
Once you have synchronized and edited the tracks, export the final video. Choose the desired format and settings for your intended use.

Please note that the exact steps may vary depending on the video editing software you are using. Additionally, it's crucial to respect copyright and licensing agreements if you are using someone else's content in your videos.

**Reply by SanjitRaman** [reply]
Thank you for your reply, so to be clear, are you suggesting that one should create one file with the two videos synchronised, and horizontally stacked for example?
